test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/Infrastructure/DthTestServer.cs

```diff
@@ -19,12 +19,12 @@ public class DthTestServer : IDisposable
         private const string DthName = "Microsoft.Dnx.DesignTimeHost";
         private Process _process;
 
-        public static DthTestServer Create(string runtimeHomePath, string projectPath, TimeSpan timeout)
+        public static DthTestServer Create(DnxSdk sdk, string projectPath, TimeSpan timeout)
```

Comment:
Exactly. @JunTaoLuo and I actually discussed about this today. It's definitely doable. But not a high priority at this point. Once we wrapped up beta8 we can revisit this topic.